src/hbbft.erl

```diff
@@ -1,27 +1,24 @@
 -module(hbbft).
 
--export([init/5, input/2, finalize_round/3, next_round/1, get_encrypted_key/2, decrypt/2, handle_msg/3]).
-
--record(hbbft_data, {
-          batch_size :: pos_integer(),
-          secret_key :: tpke_privkey:privkey(),
-          n :: pos_integer(),
-          f :: pos_integer(),
-          j :: non_neg_integer(),
-          round = 0 :: non_neg_integer(),
-          buf = [] :: [binary()],
-          acs :: hbbft_acs:acs_data(),
-          acs_init = false :: boolean(),
-          sent_txns = false :: boolean(),
-          sent_sig = false :: boolean(),
-          acs_results = [] :: [{non_neg_integer(), binary()}],
-          dec_shares = #{} :: #{non_neg_integer() => {non_neg_integer(), erlang_pbc:element()}},
-          decrypted = #{} :: #{non_neg_integer() => [binary()]},
-          sig_shares = #{} :: #{non_neg_integer() => {non_neg_integer(), erlang_pbc:element()}},
-          thingtosign :: undefined | erlang_pbc:element()
-         }).
+-include_lib("hbbft.hrl").
```

Comment:
Do these need to be header files?

Reply:
Also, if they do need to stay headers, maybe you should use `-include`, not `-include_lib`?

Reply:
removed in e728646